portal/src/texts/profile/picture.md

```diff
@@ -1,5 +1,288 @@
+
+[bilde: 3 x zoom]
+
+Når vi står på bakkeplan sammen med menneskene passer vi på at vi står vinkelrett mot bakgrunnen og danner et ettpunktsperspektiv. Horisontlinjen skal være vannrett, mens eventuelle mennesker eller gjenstander kan gjerne bryte opp det stramme uttrykket. (Wes Andreson og Stanley Cubrick er kjent for sin ekstreme bruk av ettpunktsperspektivet). 
+
+[bilde: Verksteddama og portrett fra Brandpad]
+
+Bildene skal brukes på responsive nettsider, og både format og størrelse kan bli endret, samtidig som andre elementer kan endre plassering i forhold til bildet. Med godt med rom rundt hovedmotivet i bildet kan man enklere plassere designelementer over bildet og man har et sikkerhetsrom rundt hovedmotivet slik at hovedmotivet ikke blir beskåret ved skalering.
+
+[animasjon: Vise samme bilde på flere flater med  og uten luft]
+
+Bildets format skal være rektangulært. Det skal ikke beskjæres så det blir for smalt. Minimums høyde- og breddeforhold er 2:35:1 (cinemascope) Vi legger aldri masker over bildene slik at de har andre geometriske former og vi fyller ikke objekter eller typografi med fotofyll. Hovedmotivet skal ikke beskjæres.
+
+[bilde: Minste format og for smalt format + rund ramme + typografi med fyll]
+
+Tenk kontraster og balanse når du setter opp flere bilder sammen. I et oppslag bør vi bruke bilder med varierte farger, motiv og komposisjon. Bruk en blanding av bilder tatt rett på og rett ovenfra, vis forskjellige typer mennekser, med ulik plassering i bildene.
+
+[bilde: Collage god - kanskje gif, med ulike versjoner]
+
+Når vi tar portretter av mennesker, dyr eller detaljbilder er det fint med liten dybdeskarphet somgjør bakgrunnen uklar og løfter motivet . Mens når vi tar dronebilder, landskapsbilder og oversiktsbilder er det ønskelig at hele motivet er skarpt.
+
+[bilde: nærbilde og oversiktsbilde]
+
+	•	På toppen av tinden har oversiktsbilder i tre nivåer
+	•	På bakkeplan holder vi oss til ettpunktsperspektivet
+	•	Ha rom rundt hovedmotivet til beskjæring og overlapping
+	•	Formatet er rektangulært og ikke for smalt
+	•	Tenk balanse og kontrast når du setter sammen flere bilder
+	•	Portretter har mindre dybdeskarphet enn oversiktsbilder
+
+## Bevegelse
+Bilder og bevegelse: Parallax, blureffekt, elevasjon/overlapping
+
+Oversiktsbildene fra toppen av tinden gjør seg godt som korte filmer eller cinemagrafer (stillbilde med små repeterande bevegelser).
+
+Noter deg at stilen med bilder tatt i fotostudio med helsort eller helhvit bakgrunn er forbeholdt bilder i bevegelse.
+
```

Comment:
Cinemagraphs følger i utgangspunktet samme prinsipper som for stillbilder. 

For cinemagraphs, film og dronefilm kameraet ha bør ha en fast posisjon i forhold til motivet enten vertikalt (90 grader ned) eller horisontalt (rett på  motivet). Kamera beveger seg i en rett, jevn linje slik at det enten følger følger en bevegelse i hovedmotivet eller at den rette linjen i kamerabevegelsen skaper dynamikk til motivet. Kamera kan også ha et fast standpunkt hvor elementer i motivet har en diskret bevegelse/lys som blinker/skyer i bevegelse. Opptakene må være stødige og «gimbaled»